src/SDL/vulkan.cs

```
@@ -47,12 +49,22 @@ public static partial class SDL
 		/// </summary>
 		/// <param name="path">The platform dependent Vulkan loader library name, or NULL.</param>
 		[DllImport(nativeLibName, EntryPoint = "SDL_Vulkan_LoadLibrary", CallingConvention = CallingConvention.Cdecl)]
-		private static extern int INTERNAL_Vulkan_LoadLibrary(
+        private static extern int INTERNAL_Vulkan_LoadLibrary(
 			byte[] path
 		);
-		public static int Vulkan_LoadLibrary(string path)
-		{
-			return INTERNAL_Vulkan_LoadLibrary(
+
+        [SdlAvailable("2.0.6")]
+		public static int Vulkan_LoadLibrary(string path) {
+			
+            // This is a thought. Perhaps there is another way to handle this?
+			// I'm not really great with Attribute handling.
+			// @Blizzardo1
+
+            var sdl = (SdlAvailableAttribute)MethodBase.GetCurrentMethod()
+                .GetCustomAttributes(typeof(SdlAvailableAttribute), false).FirstOrDefault();
+            if (sdl != null && !sdl.Equals()) return -1;
```

Yeah, the syntax for attributes is a little ugly. I'd write something like 

```csharp
var sdl = typeof(Whatever)
    .GetMethod(nameof(Something))
    .GetAttribute<SdlAvailableAttribute>();
if(sdl?.Equal() != true) // whatever
```

But I'd also just throw an exception instead of returning `-1` here. I believe it is more intuitive to get the exception here instead of getting a MethodNotFoundException or similar

---

Yeah this also was removed. Let me know if you would like it reimplemented.